### PR TITLE
chore(storybook): add missing 'skin/utility' and fix type imports

### PR DESCRIPTION
**Pull request by HenriqueLimas**
## Description

- Add `skin/utility` class on storybook
- Add `import type` to `global.ts` imports

## Context

Without `skin/utility` the `clipped` class is not styled properly. 
WIthout `import type`, the following error is show on storybook locally:

image

## Screenshots

Without skin/utility

image

With skin/utility

Screen Shot 2024-01-04 at 11 06 46 AM

### CODE REVIEW

File: .storybook/preview.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import "./custom-styles.less";
 import "@ebay/skin/tokens";
 import "@ebay/skin/global";
 import "@ebay/skin/marketsans";
+import "@ebay/skin/utility";
```
Comment (agliga) [inline .storybook/preview.ts:5]: Lets remove this, change `@ebay/skin/global` to `@ebay/skin/core`
Core imports both global and utility.

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
         "tb": "npm run test:browser",
         "tf": "npm run test:fps",
         "tr": "npm run coverage && npm run report && open .coverage/index.html",
-        "storybook": "storybook -p 6006",
+        "storybook": "storybook dev -p 6006",
```
Comment (agliga) [inline package.json:36]: Why was this changed? Just curious

**Review by agliga — COMMENTED** [review]
LGTM, two comments (one nit and another question)

File: package.json (hunk 2 above)
Comment (HenriqueLimas) [reply package.json:36]: It is part of storybook 7 upgrade. The same is used on `npm start` so maybe this script is not needed

**Review by agliga — APPROVED** [review]

**Review by LuLaValva — APPROVED** [review]
